src/OrchardCore/OrchardCore.DisplayManagement/Extensions/RazorHelperExtensions.cs

```diff
@@ -14,6 +14,14 @@ public static string CultureDir(this IOrchardHelper _)
         return CultureInfo.CurrentUICulture.GetLanguageDirection();
     }
 
+    /// <summary>
+    /// Gets whether the culture is RTL or not.
+    /// </summary>
+    public static bool IsRightToLeft(this IOrchardHelper _)
```

Comment:
No need use

https://github.com/OrchardCMS/OrchardCore/blob/48b350b9a6c2986a3ffcf4b37c59a7bd3f58dc5c/src/OrchardCore/OrchardCore.Localization.Abstractions/Extensions/CultureInfoExtensions.cs#L22C24-L22C37

Reply:
@hishamco 
This is as helpful as using the existing `CultureDir()`  or `CultureName()`. Using the `IsRightToLeft()` accessing the `IsRightToLeft()` method directly will require you to use to additional using statements in every view.

Reply:
You can use `CultureDir() == LanguageDirection.RTL` instead of introducing a new API that wraps another one

Reply:
I rather keep it simple and use the `IsRightToLeft()` on IOrchardHelper

Reply:
Let us see @sebastienros thought

Reply:
@MikeAlhayek I think you were ignoring this comment and merging the PR without waiting for others' feedback which is not good, especially in OSS